cirrus/server/cirrus/main.py

```diff
@@ -137,10 +169,17 @@ async def compute_features(request_data: FeatureRequest):
     enrolled_partial_configuration: Dict[str, Any] = app.state.sdk.compute_enrollments(
         targeting_context
     )
+    print(enrolled_partial_configuration)
```

Comment:
This is totally empty, but I'd expect to see events for the recipes even if the client isn't enrolled in any.

Reply:
I'd probably start with writing tests for the first two cases:`if not request_data.client_id` and `if not request_data.context`. Is the `targeting_context` being created correctly?

Reply:
Oh so sorry, this isn't actually what I'm testing. I'm testing to check that this produces the telemetry we expect to see